### PR TITLE
Add automation for needinfo requests

### PR DESCRIPTION
**Pull request by kewisch**
I think it would be useful to have some automation to close stale issues if the reporter does not respond. Here is what I have been using personally.

Also tagging @madhens in case the wording needs to be adjusted.

### CODE REVIEW

File: .github/workflows/needinfo-stale.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close old issues with the needinfo tag
+        uses: dwieeb/needs-reply@v2
```
Comment (wmontwe) [inline .github/workflows/needinfo-stale.yml:14]: This is not actively developed anymore, but I've seen that you created a pull-request. Should we wait until that's approved or use a fork?

**Review by wmontwe — APPROVED** [review]
LGTM

File: .github/workflows/needinfo-stale.yml (hunk 1 above)
Comment (cketti) [inline .github/workflows/needinfo-stale.yml:14]: Let's explicitly set `days-before-close` so we don't have to check the action's documentation to figure out when an issue is considered old.

Comment (kewisch) [reply .github/workflows/needinfo-stale.yml:14]: Done, thanks!

**Pull request merged**